.gitignore

```diff
@@ -179,3 +179,13 @@ test-suite/.nra.cache
 
 plugins/ssr/ssrparser.ml
 plugins/ssr/ssrvernac.ml
+
+# Files automatically generated by Dune.
+META.coq
+.merlin
+ide/.merlin
```

Comment:
I don't think it makes sense to ignore those files while they're still in the repo.

Reply:
I think that `configure` is the wrong tool for that; first, because I cannot properly keep track of dependencies, second, because it is also used by Dune.

Reply:
For example `META.coq` will contain version and branch information and we shouldn't have to reconfigure to update it, similarly for merlin, etc...

Reply:
See #8054